deploy/gk-deploy

```diff
@@ -499,3 +501,52 @@ else
 fi
 
 output "Ready to create and provide GlusterFS volumes."
```

Comment:
Ok then move it down when heketi is ready. This is displayed when deploy heketi is started successfully.